production/loki-mixin/dashboards.libsonnet

```diff
@@ -4,9 +4,9 @@
 (import 'dashboards/loki-logs.libsonnet') +
 (import 'dashboards/loki-operational.libsonnet') +
 (import 'dashboards/loki-reads.libsonnet') +
+(import 'dashboards/loki-reads-resources.libsonnet') +
```

Comment:
did you mean to change this, or was it a linter change?

Reply:
I did, I thought having both read dashboards close to each other made sense but I can revert this one of you prefer